### DISCUSSION TITLE
闪退的问题解决后，无法进入pro付费版，以前的激活码也无法使用了。请重发激活码

### DISCUSSION THREAD

**Discussion by rawmin110022** [getlantern/lantern-forum-cn#389] [Q&A] [answered]
闪退的问题解决后，无法进入pro付费版，以前的激活码也无法使用了。请重发激活码

rowmind@icloud.com

**Comment by MaxMKMax** [comment]
@rawmin110022 嗯嗯好的. 我们会重新发激活码给您. 请等一等. 谢谢.
请留意邮件或者GitHub, 我们会发给您.
谢谢您的耐心等待, 也谢谢您支持蓝灯!

**Comment by rawmin110022** [comment]
好的，谢谢，方便请发邮件。（尚未收到）

**Reply by MaxMKMax** [reply]
@rawmin110022 
你好。您可以尝试检查您的电子邮件是否有可用于登录的新激活码?

**Comment by rawmin110022** [comment, marked as answer]
问题以解决，管理员回复方法。thx

大致意思是：这种状态是由于当前设备已经logout，可以使用菜单里的“Authorize Device”，通过注册时候的邮箱即可重新login，问题就解决了。大家可参考。
